### PR TITLE
Modification and New Addition of Links

### PR DESCRIPTION
**Pull request by rollingstones**
Modified some links to reach the specific page and added some more links
to programming contest and problem set.

### CODE REVIEW

**Comment by rollingstones** [comment]
What went wrong?

File: problem-sets-competitive-programming.md [hunk 1]
```diff
@@ -7,12 +7,13 @@
 
 ### Competitive Programming
 
+* [A2 Online Judge](https://a2oj.com)
```
Comment (vhf) [inline problem-sets-competitive-programming.md]: ||A|| comes after ||4|| ;)

**Comment by rollingstones** [comment]
Aw, haha.

**Comment by vhf** [comment]
Thanks!

**Pull request merged**